shared/charts/gpii-preferences/values.yaml

```diff
@@ -10,7 +10,11 @@ datasourceListenPort: 5984
 datasourceHostname: "http://admin:password@couchdb-svc-couchdb.gpii.svc.cluster.local"
 
 nodeEnv: gpii.config.preferencesServer.standalone.production
+
 enableStackdriverTrace: false
+## When Stackdriver Trace is enabled SA credentials must be provided in a secret,
```

Comment:
Why double `#` comments?

Reply:
Double `#` helps to distinguish between actual comments and commented configuration blocks. It follows Helm style convention:
https://github.com/gpii-ops/gpii-infra/blob/master/shared/charts/couchdb/values.yaml
https://github.com/gpii-ops/gpii-infra/blob/master/shared/charts/nginx-ingress/values.yaml

Reply:
This seems pretty weird to me, I would still prefer single comments, but I can live with that :)